generic/tests/vlan.py

```diff
@@ -141,6 +142,34 @@ def flood_ping(src, dst):
                         session=session_flood, timeout=10)
         session_flood.close()
 
+    def get_netkvmco_path(session):
+        """
+        Get the proper netkvmco.dll path from iso.
+
+        :param session: a session to send cmd
+        :return: the proper netkvmco.dll path
+        """
+
+        viowin_ltr = virtio_win.drive_letter_iso(session)
+        if not viowin_ltr:
+            err = "Could not find virtio-win drive in guest"
+            test.error(err)
+        guest_name = virtio_win.product_dirname_iso(session)
+        if not guest_name:
+            err = "Could not get product dirname of the vm"
+            test.error(err)
+        guest_arch = virtio_win.arch_dirname_iso(session)
+        if not guest_arch:
+            err = "Could not get architecture dirname of the vm"
+            test.error(err)
+
+        middle_path = "%s\\%s" % (guest_name, guest_arch)
+        find_cmd = 'dir /b /s %s\\netkvmco.dll | findstr "\\%s\\\\"'
+        find_cmd %= (viowin_ltr,  middle_path)
```

Comment:
If I were writing this, I will just format once.
`find_cmd = dir /b /s %s\\netkvmco.dll | findstr "\\%s\\%s\\\\" % (viowin_ltr, guest_name, guest_arch)`
But you can keep it this way